modulemd/v2/tests/ModulemdTests/translation_helpers.py

```diff
@@ -68,6 +68,52 @@ def test_translation_catalog_from_index(self):
         # There are 34 unique summaries and descriptions in f29.yaml
         self.assertEquals(len(catalog), 34)
 
+    def test_translations_from_catalog(self):
+        translation_files = ["../test_data/nl.po"]
```

Comment:
This will only work if the current working directory is the same as the file location. Look at https://github.com/fedora-modularity/libmodulemd/blob/master/modulemd/v2/tests/ModulemdTests/modulestream.py#L1111 for an example of how to specify the test data location relative to the `$MESON_SOURCE_ROOT`.